scripts/libs/external-config.js

```diff
@@ -16,9 +16,9 @@ export const externalConfig = {
 
   async loadConfig() {
     window.raqnComponentsConfig ??= (async () => {
-      const { metaName, fallbackContent } = metaTags.themecomponent;
-      const metaConfigPath = getMeta(metaName);
-      const configPath = (!!metaConfigPath && `${metaConfigPath}.json`) || `${fallbackContent}.json`;
+      const { metaName } = metaTags.themeConfigComponent;
+      const metaConfigPath = getMeta(metaName, { getFallback: true });
```

Comment:
Should we have the fallback returned as default,
Seems needless to ask for it.

Reply:
I agree, I will change it